README.md

```diff
@@ -1,94 +1,87 @@
+# including `tox`, a LXD / Kubernetes cloud -- 'localhost' / 'microk8s' -- and Juju,
+# then use Juju to bootstrap a controller into a cloud -- below, 'microk8s'-
+# and add a model that you'll deploy your charm to.
+sudo snap install multipass && \
+multipass launch --cpus 4 --memory 8G --disk 30G --name tutorial-vm charm-dev
+juju bootstrap microk8s tutorial-controller; juju add-model tutorial-model
 
-The framework provides a standardised Python object model that represents the
-application graph, as well as an event-handling mechanism for distributed
-system coordination and communication.
+# Create a charm directory and use Charmcraft to initialise your charm file structure:
+mkdir my-new-charm; cd my-new-charm; charmcraft init
 
-The latest version of ops requires Python 3.8 or above.
+# The files are already populated with functional templates. Poke around. 
 
-Juju itself is written in Go for efficient concurrency even in large
-deployments. Charms can be written in any language, however, we recommend using
-Python with this framework to make development easier and more standardised.
-All new charms at Canonical are written using it.
+# Note that the ‘metadata.yaml’ shows that what we have is an example charm called ‘my-new-charm’
+# which uses an OCI image resource ‘httpbin’ from ‘kennethreitz/httpbin’.
 
+# Note that the ‘requirements.txt’ file already lists a version of ‘ops’. 
 
-## Getting started
+# Note that ‘src/charm.py’ already uses ops constructs
+# to create a charm class ‘MyNewCharmCharm’, observe Juju events, and pair them to event handlers:
+#
+# class MyNewCharmCharm(ops.CharmBase):
+#     """Charm the service."""
+# 
+#     def __init__(self, *args):
+#         super().__init__(*args)
+#         self.framework.observe(self.on['httpbin'].pebble_ready, self._on_httpbin_pebble_ready)
+#         self.framework.observe(self.on.config_changed, self._on_config_changed)
+# ...
 
-A package of operator code is called a charmed operator or simply "charm".
-You'll use [charmcraft](https://juju.is/docs/sdk/install-charmcraft) to
-register your charm name and publish it when you are ready. You can follow one
-of our [charming tutorials](https://juju.is/docs/sdk/tutorials) to get started
-writing your first charm.
+# Edit the files to make the charm your own.
 
+# When you’re done, pack the example charm into a .charm file:
+charmcraft pack
 
-## Testing your charms
+# If you haven’t edited the name of the charm,
+# this will yield a file called ‘my-new-charm_ubuntu-22.04-amd64.charm’
+# (the architecture bit may be different depending on your system's architecture).
 
-The framework provides a testing harness, so you can ensure that your charm
-does the right thing in different scenarios, without having to create
-a full deployment. Our [API documentation](https://ops.readthedocs.io/en/latest/#module-ops.testing)
-has the details, including this example:
```

Comment:
We definitely do not want to remove mention of testing, or the example. (We could potentially go further the other way, mentioning pytest-operator and scenario, but it makes sense to wait until the discussions in Riga around testing direction).